core-services/prow/02_config/_plugins.yaml

```diff
@@ -2,6 +2,7 @@ blunderbuss:
   max_request_count: 2
   request_count: 2
   use_status_availability: true
+branch_cleaner: {}
```

Comment:
Looks like the main branch has changed since the time that the I pushed this PR. Will need to `git pull upstream master` and `make update` just to be sure everything is good.

Reply:
I pulled in the latest upstream master. Let's see if this works